src/Form.js

```diff
@@ -0,0 +1,52 @@
+import React, {memo, useState} from 'react';
+import styled from "styled-components";
+
+export const StyledForm = styled.form`
+    display: flex;
+    width: 270px;
+    white-space: nowrap;
+    align-items: center;
+    align-self: flex-end;
+`
+export const Input = styled.input`
+    width: 70vw;
+    margin: 10px;
+    margin-left: 0px;
+    height: 30px;
+    border: solid 1px #dcdcdc;
+    border-radius: 10%;
+`
+export const Button = styled.button`
+    width: 10vw;
+    height: 35px;
+    background: yellow;
+    border: 0;
+    border-radius: 20px;
+    margin-right: 5px;
+`
+const Form = memo(({ isMe, upLoadChatting, chatting }) => {
```

Comment:
```suggestion
const Form = memo(({ isMe, uploadChatting, chatting }) => {
```

사소한 것이지만 upload는 한 단어니까 굳이 중간에 대문자를 넣지 않아도 될 것 같습니다.